yb-voyager/cmd/importDataToSourceReplica.go

```diff
@@ -60,11 +60,20 @@ func init() {
 	registerCommonGlobalFlags(importDataToSourceReplicaCmd)
 	registerCommonImportFlags(importDataToSourceReplicaCmd)
 	registerSourceReplicaDBAsTargetConnFlags(importDataToSourceReplicaCmd)
+	registerFlagsForSourceReplica(importDataToSourceReplicaCmd)
+	registerStartCleanFlag(importDataToSourceReplicaCmd)
 	registerImportDataCommonFlags(importDataToSourceReplicaCmd)
-	registerImportDataFlags(importDataToSourceReplicaCmd)
 	hideImportFlagsInFallForwardOrBackCmds(importDataToSourceReplicaCmd)
 }
 
+func registerStartCleanFlag(cmd *cobra.Command) {
+	BoolVar(cmd.Flags(), &startClean, "start-clean", false,
+		`Starts a fresh import with exported data files present in the export-dir/data directory. 
+If any table on source-replica database is non-empty, it prompts whether you want to continue the import without truncating those tables; 
+If you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files without upsert mode.
```

Comment:
Should this be: _"... data files with upsert mode"_

Reply:
As discussed, this should be without upsert mode for source-replica.